crates/qt-build-utils/Cargo.toml

```diff
@@ -14,7 +14,7 @@ repository = "https://github.com/KDAB/cxx-qt/"
 
 [dependencies]
 versions = "4.1.0"
-pkg-config = { git = "https://github.com/Be-ing/pkg-config-rs.git", branch = "library_pub" }
```

Comment:
this won't work as it still requires vendoring of some private methods, but now that upstream has stabilised what we are going to use we can implement this.

Reply:
Ah, alright, I thought the changes linked in the pkg-config-rs PR were enough for our purposes :thinking:

Reply:
I will work on decoupling the required code from pkg-config's structs this week.

Reply:
For context: I initially tried to use pkg-config, but [Qt6 doesn't ship .pc files for static builds yet](https://bugreports.qt.io/browse/QTBUG-86080). Also, using pkg-config would require somehow getting the pkg-config executable on Windows. Instead, we're parsing the .prl files which are used by QMake.